src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -163,25 +163,40 @@ def send_incident_alert_notification(
     return success
 
 
+class SlackCommand(Enum):
+    LINK = "link"
+    UNLINK = "unlink"
+
+
 def respond_to_slack_command(
-    params: IdentityParams,
-    text: str,
-    command: str,
+    command: SlackCommand,
+    integration: Integration,
+    slack_id: str,
+    response_url: str | None,
 ) -> None:
-    log = "slack.link-identity." if command == "link" else "slack.unlink-identity."
+    from sentry.integrations.slack import SUCCESS_LINKED_MESSAGE, SUCCESS_UNLINKED_MESSAGE
+
+    if command is SlackCommand.LINK:
+        log = "slack.link-identity."
+        text = SUCCESS_LINKED_MESSAGE
+    elif command is SlackCommand.UNLINK:
+        log = "slack.unlink-identity."
+        text = SUCCESS_UNLINKED_MESSAGE
+    else:
+        raise ValueError
 
-    if params.response_url:
-        logger.info(log + "respond-webhook", extra={"response_url": params.response_url})
+    if response_url:
+        logger.info(log + "respond-webhook", extra={"response_url": response_url})
         try:
-            webhook_client = WebhookClient(params.response_url)
+            webhook_client = WebhookClient(response_url)
             webhook_client.send(text=text, replace_original=False, response_type="ephemeral")
             metrics.incr(
                 SLACK_LINK_IDENTITY_MSG_SUCCESS_DATADOG_METRIC,
                 sample_rate=1.0,
                 tags={"type": "webhook", "command": command},
             )
         except (SlackApiError, SlackRequestError) as e:
-            if unpack_slack_api_error(e) != EXPIRED_URL:
+            if "Expired url" not in str(e):
```

Comment:
@RyanSkonnord should this be a constant?

Reply:
Oh, I didn't do this on purpose. This was how it was in an earlier revision -- I must have goofed when I was diffing against Git history. I'll revert.